courses/fundamentals_of_ada/190_exceptions.rst

```diff
@@ -762,6 +762,23 @@ Exception Occurence
       + Location where exception occurred
       + Language-defined check that failed (if such)
 
+--------------
+Exception ID
+--------------
+
+* For an exception identifier, the :defn:`identity` of the exception is :ada:`<name>'Identity`
```

Comment:
Definitions are done with role `:dfn:`, not `:defn:`. None are supported by pandoc for the moment but it would be good to keep the sphinx-portable notation.